program-test/src/programs.rs

```diff
@@ -4,15 +4,15 @@ mod spl_token {
     solana_sdk::declare_id!("TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA");
 }
 mod spl_memo {
-    solana_sdk::declare_id!("Memo1UhkJRfHyvLMcVucJwxXeuD728EqVDDwQDxFMNo");
+    solana_sdk::declare_id!("MemoSq4gqABAXKb96qnH8TysNcWxMyWCqXgDLGmfcHr");
 }
 mod spl_associated_token_account {
     solana_sdk::declare_id!("ATokenGPvbdGVxr1b2hvZbsiqW5xWH25efTNsLJA8knL");
 }
 
 static SPL_PROGRAMS: &[(Pubkey, &[u8])] = &[
     (spl_token::ID, include_bytes!("programs/spl_token-3.1.0.so")),
-    (spl_memo::ID, include_bytes!("programs/spl_memo-1.0.0.so")),
+    (spl_memo::ID, include_bytes!("programs/spl_memo-3.0.0.so")),
```

Comment:
The new SO needs to be added to the repo as well in here: https://github.com/solana-labs/solana/tree/master/program-test/src/programs

Reply:
Ah thanks, for some reason I thought this picked up from fetch-spl

Reply:
Would be nice :)
The problem is when the program-test crate is published to crates.io, ` fetch-spl.sh` is no longer a thing